config/default/configmap/kfservice.yaml

```diff
@@ -39,3 +39,8 @@ data:
            "s3SecretAccessKeyName": "awsSecretAccessKey"
        }
     }
+  ingress: |-
+    {
+        "ingressGateway" : "knative-ingress-gateway.knative-serving",
```

Comment:
Do we want to make our own kfserving gateway?

kfservice.kfserving

Reply:
Gateway is configurable in knative... the default is `knative-ingress-gateway`. See https://github.com/knative/serving/releases/download/v0.9.0/serving.yaml

I dont think we should create our own gateway (unless we are planning on a custom knative deployment)